src/plugins_3rdparty/umuscle/src/MuscleParallel.cpp

```diff
@@ -64,16 +64,18 @@ int MuscleParallelTask::estimateMemoryUsageInMb(const MultipleSequenceAlignment
     }
     std::sort(rowsLengths.begin(), rowsLengths.end(), std::greater<int>());
 
-    qint64 usedBytes = 0;
+    quint64 usedBytes = 0;
```

Comment:
Don't use unsigned types. It gives more problem than solutions

Reply:
What kind of problem? I think more naturally to use unsigned because memory consumption value can't be negative value.

Reply:
See the problem in your code below. 
As another example you can guess other popular modern languages that separate signed and unsigned type. I think this is an artifact inherited from the old low-level C language and it should be avoided to have no extra type conversions in the code. 
Another example: see QT collections or strings: they all use signed types for size/capacity, etc.

Reply:
quint64 is back